### ISSUE TITLE
Got my lvl 70 boost x2, But no gold??!!

### ISSUE THREAD

**Issue by Rotteknuser** [dalaranwow/dalaran-wow#4268]
![skjermbilde 2017-01-19 13 20 07](https://cloud.githubusercontent.com/assets/25211969/22106648/0acbc4ea-de4a-11e6-9bd1-edfd71984253.png)
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: One Druid (Noqqer) and one Shaman (Rotteknuser) Level 70 in Ogrimar after lvl 70 boost with no 900 gold as part og the deal. 

**Expected behaviour**: I expected to have

**Steps to reproduce the problem**:

1. Bye Dp.
2. Select simple boost; 900 gold, lvl 70 and simple green gear.
3. Did only get the boost... no gear and no gold. No gear is ok, Since i can go in AH and get some, but No gold meens no talent skills, whivh means leveling takes a loot longer. 

**Include proofs for this behaviour**: Considering you should be able to know what i bought and what I did not get in the transaction you should sit with all the proof your self.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
![skjermbilde 2017-01-19 13 13 26](https://cloud.githubusercontent.com/assets/25211969/22106520/5416d208-de49-11e6-9476-620b17dc7aef.png)

Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary** I bought two services were  i got level 70x2 but not 900 gold x2 (im not able to train my respected talents to do anything).

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
